system/datatypes/festival.php

```diff
@@ -97,6 +97,11 @@ function process(HyphaRequest $request) {
 			}
 		}
 
+		public function getSortDateTime() {
+			// TODO: What to return here?
```

Comment:
Is this WIP or left for a later time/issue?

Reply:
Left for later indeed.